src/components/SpotsMap/component.tsx

```diff
@@ -28,6 +29,10 @@ export const SpotsMap: React.FC<SpotsMapProps> = () => {
     setCenter(location);
   }, [location, center]);
 
+  useEffect(() => {
+    (listRef.current as HTMLElement).scrollLeft = 0;
```

Comment:
The type guard is needed because it can potentially be `null`.

```suggestion
    if (listRef.current) {
      listRef.current.scrollLeft = 0;
    }
```